FWCore/Skeletons/scripts/mkTemplates/EDAnalyzer/EDAnalyzer.cc

```diff
@@ -105,23 +104,23 @@ __class__::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 {
    using namespace edm;
 
-@example_track   using reco::TrackCollection;
+   using reco::TrackCollection;
 
-@example_track    Handle<TrackCollection> tracks;
-@example_track    iEvent.getByLabel(trackTags_,tracks);
-@example_track    for(TrackCollection::const_iterator itTrack = tracks->begin();
-@example_track        itTrack != tracks->end();                      
-@example_track        ++itTrack) {
-@example_track       int charge = 0;
-@example_track       charge = itTrack->charge();  
-@example_histo       histo->Fill( charge );
-@example_track    }
+    Handle<TrackCollection> tracks;
+    iEvent.getByLabel(trackTags_,tracks);
```

Comment:
We should really change this to be a `getByToken` instead.

Reply:
Indeed, thanks, I blatantly did cut and paste from old code. Will fix it.